packages/edit-site/src/components/layout/index.js

```diff
@@ -183,16 +183,18 @@ export default function Layout( { route } ) {
 							</div>
 						) }
 
-					{ ! isMobileViewport && areas.edit && (
-						<div
-							className="edit-site-layout__area"
-							style={ {
-								maxWidth: widths?.edit,
-							} }
-						>
-							{ areas.edit }
-						</div>
-					) }
+					{ ! isMobileViewport &&
+						areas.edit &&
+						canvasMode !== 'edit' && (
```

Comment:
What's the reasoning for this. I mean, it's ok, but feels like maybe it's the responsibility of the route no?

Reply:
The logic is the same as any other area: for example, `content`, `sidebar`, or `mobile` also have this check.

Reply:
Let's go for consistency then for now.

Reply:
This is triggering an ESLint error since `canvasMode` is undefined.

Link: https://github.com/WordPress/gutenberg/actions/runs/11458887350/job/31882230297?pr=65340#step:5:456

Reply:
Oh, interesting. I wonder why the tests didn't report any issue for this PR? I see it was fixed at https://github.com/WordPress/gutenberg/pull/66316

Reply:
I think a follow-up PR (#66213) removed the variable, but the conflict wasn't caught until it was merged.